clients/bazel-module-registry/src/main/kotlin/LocalBazelModuleRegistryService.kt

```diff
@@ -35,7 +35,7 @@ const val SOURCE_JSON = "source.json"
 /**
  * A Bazel registry which is located on the local file system.
  */
-class LocalBazelModuleRegistryService(directory: File) : BazelModuleRegistryService {
+class LocalBazelModuleRegistryService(val url: String, directory: File) : BazelModuleRegistryService {
```

Comment:
Sorry, same thing here: Drop `val`. The inspection did not trigger here as the properly is public currently.

Reply:
Also, please document the `url` and `directory` constructor parameters to explain their relation (e.g., I was wondering whether the `directory` can't be deduced from the file://-type `url`).